adapters/adform/adform_test.go

```diff
@@ -320,9 +323,9 @@ func createTestData(secure bool) aBidInfo {
 		tid:       "transaction-id",
 		buyerUID:  "user-id",
 		tags: []aTagInfo{
-			{mid: 32344, keyValues: "color:red,age:30-40", keyWords: "red,blue", priceType: "gross", code: "code1", price: 1.23, content: "banner-content1", dealId: "dealId1", creativeId: "creativeId1"},
-			{mid: 32345, priceType: "net", code: "code2"}, // no bid for ad unit
-			{mid: 32346, code: "code3", price: 1.24, content: "banner-content2", dealId: "dealId2"},
+			{mid: 32344, keyValues: "color:red,age:30-40", keyWords: "red,blue", cdims: "300x300:400x200", priceType: "gross", code: "code1", price: 1.23, content: "banner-content1", dealId: "dealId1", creativeId: "creativeId1"},
```

Comment:
The colon (:) separator doesn't work and instead a comma (,) should be used so: _cdims: "300x300:400x200"_ should become _cdims: "300x300,400x200"_